aws-toolkit-vscode.code-workspace

```diff
@@ -1,19 +1,50 @@
 {
```

Comment:
You shouldn't commit the changes to this file

Reply:
I'll remove this.